doc/api/test.md

```diff
@@ -808,6 +808,11 @@ Shorthand for skipping a suite, same as [`describe([name], { skip: true }[, fn])
 Shorthand for marking a suite as `TODO`, same as
 [`describe([name], { todo: true }[, fn])`][describe options].
 
+## `describe.only([name][, options][, fn])`
```

Comment:
If we do this, we should also support `test.only`

Reply:
`test` does not currently support `test.skip()` or `test.todo()`, and I would prefer to keep it that way.

Reply:
I think there is definitely value in having shorthands for the `test` API - especially `skip` and `only`. However, for the scope of this commit, it's only about adding the extra shorthand to the `describe/it` API. This keeps the options consistent.

@cjihrig are there any issues in particular you see with adding a shorthand API for the test interface in a future PR?

Reply:
There is no technical reason - just a desire to keep the API surface area smaller. The `Test` class in particular is important because everything else builds on top of it. And, it already supports skip, todo, and only functionality, so I would prefer to not implement another way of doing the same thing.